lib/utils.pm

```diff
@@ -433,12 +433,12 @@ sub ensure_unlocked_desktop {
                 type_string "$username";
                 save_screenshot;
             }
-            # On gnome, user may not be selected and using 'ret' is not enough in this case
-            if (check_var('DESKTOP', 'gnome') && (is_sle('15+') || is_leap('15.0+') || is_tumbleweed)) {
-                select_user_gnome($username);
+            if (!check_var('DESKTOP', 'gnome') && (is_sle('<15') || is_leap('<15.0'))) {
```

Comment:
This should be || instead of &&:
if (!check_var('DESKTOP', 'gnome') || (is_sle('<15') || is_leap('<15.0'))) {